docs/installation/centos.md

```diff
@@ -53,10 +53,10 @@ package manager.
 
 3. Add the yum repo.
 
-        $ echo | sudo tee /etc/yum.repos.d/docker.repo <<-EOF
+        $ echo | sudo tee /etc/yum.repos.d/docker.repo <<-'EOF'
```

Comment:
it is to prevent `$releasever` from being treated as a variable by the shell. https://unix.stackexchange.com/questions/68419/how-to-print-in-here-document has a nice explanation. it could also be escaped using `\$releasever` but I've always preferred the quotes way since it makes things easier to read.